contracts/multihop/src/tests/swap.rs

```diff
@@ -157,6 +157,482 @@ fn swap_three_equal_pools_no_fees() {
     assert_eq!(tokens[3].balance(&recipient), 50i128);
 }
 
+#[test]
+fn swap_single_pool_no_fees() {
+    let env = Env::default();
+
+    let admin = Address::random(&env);
+    let user = Address::random(&env);
```

Comment:
You are using `user` account only for liquidity, this as well may be done by `admin`.